apis/facebook/api/src/objects/fb_user.e

```diff
@@ -0,0 +1,223 @@
+note
+	description: "Object representing a Facebook User."
+	date: "$Date$"
+	revision: "$Revision$"
+	EIS: "name=Facebook User", "src=https://developers.facebook.com/docs/graph-api/reference/user/", "protocol=uri"
+
+class
+	FB_USER
+
+feature -- Access: Basic Properties
+
+	id:  detachable STRING
+			-- The id of this person's user account.
+
+	name: detachable STRING
+			-- The person's full name.
+			--! Predeterminded.
+
+	birthday: detachable STRING
```

Comment:
I've updated the description, with their definition: `The person's birthday. This is a fixed format string, like MM/DD/YYYY. However, people can control who can see the year they were born separately from the month and day so this string can be only the year (YYYY) or the month + day (MM/DD)`